libraries/chain/authorization_manager.cpp

```diff
@@ -158,7 +158,7 @@ namespace eosio { namespace chain {
          p.last_updated = creation_time;
          p.auth         = auth;
 
-         if (auto dm_logger = _control.get_deep_mind_logger()) {
+         if (auto dm_logger = _control.get_deep_mind_logger(false)) {
```

Comment:
We don't want to log this for dry-run transactions.

Reply:
I did not notice this version of create_permission is used in apply_eosio_updateauth. Thanks. Will change.